test/conformance/conformance_task_arena.cpp

```diff
@@ -50,6 +50,8 @@ TEST_CASE("Arena interfaces") {
         //! Attach interface
         oneapi::tbb::task_arena attached_arena{oneapi::tbb::task_arena::attach()};
```

Comment:
BTW As  `task_arena::attach` is deprecated , should this be moved out of conformance test ?

Reply:
Currently, it is still in the spec (deprecated section), so it should provided and tested.